setup.py

```diff
@@ -27,8 +27,7 @@
         'Programming Language :: Python :: 3.8'
     ],
     platforms='Windows, MacOS, *nix',
-    install_requires=['numpy>=1.15', 'pandas>=0.24', "tqdm",
-                      "pastas>=0.13", "arctic"],
+    install_requires=['numpy>=1.15', 'pandas>=0.24', "tqdm", "pastas>=0.13"],
```

Comment:
I think this should have the same version requirements for pandas as in requirements.txt

Reply:
Also, pastas should be a newer version right? There were some changes with the reading and writing of models